JetNews/app/src/main/java/com/example/jetnews/ui/article/PostContent.kt

```diff
@@ -198,8 +206,9 @@ private fun BulletParagraph(
     paragraphStyle: ParagraphStyle
 ) {
     Row {
-        Container(modifier = LayoutSize(8.dp, 8.dp)) {
-            DrawShape(shape = CircleShape, color = Color.DarkGray)
+        Box(modifier = LayoutSize(8.dp, 8.dp) +
+                DrawBackground(color = Color.DarkGray, shape = CircleShape)) {
```

Comment:
Not part of this CL, but we shouldn't be referencing hardcoded colors like this

Reply:
Swapping it out for this CL so we don't forget!